interop-tests/src/test/scala/org/apache/pekko/grpc/interop/PekkoGrpcScalaClientTester.scala

```diff
@@ -41,21 +41,29 @@ import scala.util.control.NoStackTrace
  * The same implementation is also be found as part of the 'scripted' tests at
  * /sbt-plugin/src/sbt-test/gen-scala-server/00-interop/src/test/scala/org/apache/pekko/grpc/PekkoGrpcClientTester.scala
  */
-class PekkoGrpcScalaClientTester(val settings: Settings, backend: String)(implicit system: ActorSystem)
-    extends ClientTester {
+class PekkoGrpcScalaClientTester(val settings: Settings, backend: String, testWithSslContext: Boolean)(
```

Comment:
maybe better with a trait
 SslContextProvider, and the false can be expressed by returning None.

Reply:
It is test code. And the SSL context is created inside this class. Not outside.

Reply:
When we have a use case for creating the SSL context outside - we can change then. Seems bad to overengineer test code